test/ctr.bats

```diff
@@ -957,3 +969,30 @@ function check_oci_annotation() {
 	done
 	crictl stop "$ctr_id"
 }
+
+@test "ctr children should be killed if ctr was killed" {
+	newsandbox="$TESTDIR/sandbox.json"
+	start_crio
+
+	jq '	  .linux.security_context.namespace_options.pid = 2' \
+		"$TESTDATA"/sandbox_config.json > "$newsandbox"
+
+	jq '	  .image.image = "quay.io/crio/redis:alpine"
+		| .linux.security_context.namespace_options.pid = 2
+		| .command = ["/bin/sh", "-c", "sleep 1m& exec sleep 2m"]' \
+		"$TESTDATA"/container_config.json > "$newconfig"
+
+	ctr_id=$(crictl run "$newconfig" "$newsandbox")
+	processes=$(list_all_children "$(pidof conmon)")
+
+	pid=$(runtime list -f json | jq .[].pid)
```

Comment:
"runtime" is a wrapper to call the runtime binary directly from the command line. It assumes a runc-style runtime.
This is not compatible with the containerd-shim interface. This is why the kata CI is failing.
You may want to skip this test when RUNTIME_TYPE=vm, at least until we find a shim-compatible way of getting the same information

Reply:
See #5896 for reference

Reply:
good catch, fixed!